docs/docs/30-administration/00-setup.md

```diff
@@ -70,6 +70,7 @@ services:
     depends_on:
       - woodpecker-server
     volumes:
+      - woodpecker-agent-config:/etc/woodpecker
```

Comment:
Assuming this is meant to be a local file, it should be `./woodpecker-agent-config`.

Just wanting to clarify here prior to opening a patch for it.

Reply:
well it depends, I interpreted it as named volume ... same as the example show for the database for the server

Reply:
Okay, then it should be added to the named volumes at the bottom of the example. I can send a quick change for that, just to clarify in docs

Reply:
@jolheiser sure go on, so we do have it in the v1.0.0 docks :D

Reply:
#2025